app/models/user.rb

```diff
@@ -1,9 +1,10 @@
 class User < ApplicationRecord
-  attr_accessor :remember_token, :activation_token
-  before_save :downcase_email
-  before_create :create_activation_digest
   USERS_PARAMS = %i(name email password password_confirmation).freeze
 
+  attr_accessor :remember_token, :activation_token, :reset_token
+
+  scope :sort_by_name, ->{order :name}
```

Comment:
You should write more detail```suggestion
  scope :sort_by_name, ->{order: name: :desc/:asc}
```